node/service/src/lib.rs

```diff
@@ -654,6 +661,8 @@ where
 			relay_chain_interface,
 			relay_chain_slot_duration,
 			import_queue,
+			// TODO: plumb this through CLI properly
```

Comment:
This relates to https://github.com/paritytech/cumulus/pull/963 and will earn its own Jira ticket.